Core/Transaction.cs

```diff
@@ -5,21 +5,23 @@ namespace Core
 {
 	public class Transaction : IComparable<Transaction>
 	{
-		public Transaction(string cardNumber, string appCode, DateTimeOffset timeStamp, Money amount, Money rest, string terminal, string description)
+		public Transaction(string cardNumber, string appCode, DateTimeOffset timeStamp, double amount, 
+			double rest, string terminal, string description)
 		{
 			CardNumber = cardNumber;
 			ApplicationCode = appCode;
 			TimeStamp = timeStamp;
-			Amount = amount.Amount >= 0 ? amount : new Money(-amount.Amount, amount.Currency);
-			IsIncome = amount.Amount > 0;
-			IsExpence = amount.Amount < 0;
+			Amount = amount >= 0 ? amount : -amount;
+			IsIncome = amount > 0;
+			IsExpence = amount < 0;
 			Rest = rest;
 			Terminal = terminal;
 			Description = description;
 		}
 
 		[JsonConstructor]
-		public Transaction(string cardNumber, string applicationCode, DateTimeOffset timeStamp, Money amount, bool isIncome, bool isExpence, Money rest, string terminal, string description)
+		public Transaction(string cardNumber, string applicationCode, DateTimeOffset timeStamp, double amount, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Constructor has 9 parameters, which is greater than the 7 authorized.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)